frontend/src/layout/header/UserBox.tsx

```diff
@@ -107,149 +171,189 @@ const WithFloatingMenu: React.FC<{ children: ReactElement }> = ({ children }) =>
             trigger="click"
             ariaRole="menu"
             arrowSize={12}
-            distance={0}
             viewPortMargin={12}
+            borderRadius={8}
         >
-            <FloatingMenu close={() => ref.current?.close()} />
             <FloatingTrigger>{children}</FloatingTrigger>
+            <FloatingMenu close={() => ref.current?.close()} type={type} />
```

Comment:
Is there a reason these swapped places?

Reply:
Yes, glad you asked :D
It's about the tab order. Placing the trigger after the menu would mean you have to `tab + shift` to tab into the menus. You can see that behaviour in the current deployment, for instance.

Reply:
Oh interesting! Makes sense